src/main/plots/ati_data_plots.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from src.main.util import consts
+from src.main.plots import consts as plot_consts
+from src.main.plots.plots_common import save_plot, add_fragments_length_plot, get_short_name, EVENT_DATA_COL, \
+    TIMESTAMP_COL, FRAGMENT_COL
+
+log = logging.getLogger(consts.LOGGER_NAME)
+
+
+def __create_ati_events_plot(ax: plt.axes, df: pd.DataFrame, event_data: list, event_colors: dict, title: str):
+    add_fragments_length_plot(ax, df)
+    for event_data in event_data:
+        event_data_df = df.loc[df[EVENT_DATA_COL] == event_data]
+        ax.scatter(event_data_df[TIMESTAMP_COL], event_data_df[plot_consts.FRAGMENT_LENGTH_COL],
+                   color=event_colors[event_data], s=plot_consts.LARGE_SIZE, label=event_data)
+    ax.legend(bbox_to_anchor=(1.04, 1), borderaxespad=0)
```

Comment:
Could you please add some description on the comments for different params like bbox_to_anchor and others? Like in statistics_plots file: `# x_category_order='total ascending' means: in order of increasing values in Y`. Because we use different libraries and it’s not always easy to find why exactly such parameters were used. Also, I think it would be useful in the future to quickly remember what these options mean